OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -19,9 +19,13 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 	{
 		[Desc("HitPoints")]
 		public readonly int HP = 0;
+
 		[Desc("Trigger interfaces such as AnnounceOnKill?")]
 		public readonly bool NotifyAppliedDamage = true;
 
+		[Desc("Distance above CenterPosition for targeting by weapons.")]
```

Comment:
I was considering to make this directly WVec, but felt it would complicate matters (and increase risk of bugs in damage calculation etc.) if X and Y were customizable as well.